src/Test/L1/Tasks/d9bafed4-0b18-4f58-968d-86655b4d2ce9/icon.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="24" height="24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can we avoid checking in png and svg files?

Reply:
Sure, done